bootstrap_haml_helpers.gemspec

```diff
@@ -17,10 +17,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency "rails"
 
-  s.add_dependency "bootstrap-sass", "2.3.2.2"
+  s.add_dependency "bootstrap-sass", "~> 3.4.1"
```

Comment:
Hmm I would be concerned here that this bump will bring in bootstrap 3, which I don't think our app can handle :(